packages/core/src/payment/payment-method-request-sender.ts

```diff
@@ -19,19 +20,26 @@ export default class PaymentMethodRequestSender {
                 'X-API-INTERNAL': INTERNAL_USE_ONLY,
                 ...SDK_VERSION_HEADERS,
             },
+            params: {
+                cartId,
+            },
         });
     }
 
-    loadPaymentMethod(methodId: string, { timeout }: RequestOptions = {}): Promise<Response<PaymentMethod>> {
-        const url = `/api/storefront/payments/${methodId}`;
+    loadPaymentMethod(cartId: string, methodId: string, { timeout }: RequestOptions = {}): Promise<Response<PaymentMethod>> {
+        const url = new URL(`/api/storefront/payments/${methodId}`, window.location.origin);
```

Comment:
Just want to check I don't think we need to do this since
```
const url = new URL(`/api/storefront/payments/method`, window.location.origin);
console.log(url.pathname) => /api/storefront/payments/method
```
is same as 
`/api/storefront/payments/${methodId}`

Reply:
Currently, some payment providers are passing the gateway-id with a parameter method-id as methodId like `afterpay?method=xxx`. Using class URL is to separate the parameter and pathname correctly.
Here is one sample:
https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/core/src/payment/strategies/afterpay/afterpay-payment-strategy.ts#L139